tests/testthat/test_special_orthogonal_group.R

```diff
@@ -48,3 +48,11 @@ test_that("Tests Regularize", {
 
 
 })
+
+test_that("Tests Skew Matrix From Vector", {
+  sp <- SpecialOrthogonalGroup$new(n = 3)
```

Comment:
let's call this `so3` instead of `sp`, as this is the right mathematical abbreviation.